lib/usb/usb_fx07_common.c

```diff
@@ -262,6 +262,23 @@ void stm32fx07_poll(usbd_device *usbd_dev)
 		return;
 	}
 
+	/*
+	 * There is no global interrupt flag for transmit complete.
+	 * The XFRC bit must be checked in each OTG_DIEPINT(x).
+	 */
+	for (i = 0; i < 4; i++) { /* Iterate over endpoints. */
+		if (REBASE(OTG_DIEPINT(i)) & OTG_DIEPINTX_XFRC) {
+			/* Transfer complete. */
+			if (usbd_dev->user_callback_ctr[i]
+						       [USB_TRANSACTION_IN]) {
```

Comment:
I'd rather not change this, since it is only moving the code from below without any changes.